docs/reference/index.md

```diff
@@ -1,143 +1,145 @@
 ---
 layout: LandingPage
```

Comment:
Is this file referenced anywhere? It seems incomplete, but if you're not using it, that makes sense :-)

Reply:
Good catch. It's not being used anywhere now -- this reference landing page ended up not being needed since the fusion TOC exposes API reference nodes in the TOC.